sovryn-token-bridge/federator/src/lib/GasPriceAvg.js

```diff
@@ -0,0 +1,20 @@
+
+module.exports = class GasPriceAvg {
+    constructor() {
+        this.arr = [90];
+    }
+ 
+    async calcAvg(_avgGasCount, elem) {
```

Comment:
This method does not need to be async since it doesn't do any `await`s (e.g. web3 calls) inside it.

Reply:
Fixed